src/server/routes/reset.ts

```diff
@@ -11,11 +11,19 @@ import { PageTitle } from '@/shared/model/PageTitle';
 import { handleAsyncErrors } from '@/server/lib/expressWrappers';
 import { readEmailCookie } from '@/server/lib/emailCookie';
 import { setEncryptedStateCookie } from '../lib/encryptedStateCookie';
-import { ResetPasswordErrors } from '@/shared/model/Errors';
+import { CaptchaErrors, ResetPasswordErrors } from '@/shared/model/Errors';
 import { ApiError } from '@/server/models/Error';
 import { addQueryParamsToPath } from '@/shared/lib/queryParams';
+import { RecaptchaV2 } from 'express-recaptcha';
+import { getConfiguration } from '@/server/lib/getConfiguration';
 
 const router = Router();
+const {
+  googleRecaptcha: { secretKey, siteKey },
+} = getConfiguration();
+
+// set google recaptcha site key
+const recaptcha = new RecaptchaV2(siteKey, secretKey);
```

Comment:
rather than having to initialise this in every file, could you make a file in `lib` with method that could export the `recaptcha` const?

Reply:
https://github.com/guardian/gateway/pull/1176#discussion_r757529383